### PR TITLE
4.13-RNs:creates 4.13 .adoc file in main

### PR DESCRIPTION
**Pull request by JoeAldinger**
4.13-RNs: creates the stub file in main

Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-4837

Link to docs preview:
https://54888--docspreview.netlify.app/openshift-enterprise/latest/welcome/index.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
@JoeAldinger just a couple of suggestions for your review...otherwise, this looks good!

/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[id="ocp-4-13-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+Do not add or edit release notes here. Edit release notes directly in the branch
+that they are relevant for.
+
+Release note changes should be added/edited in their own PR.
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-13-release-notes.adoc:9]: I think this is just a RN stock text thing. This doesn't appear anywhere in our documentation. It is only visible in the repo docs.

**Review by opayne1 — COMMENTED** [review]
This looks good to me!

**Pull request merged**